rust/kcl-lib/src/parsing/parser.rs

```diff
@@ -2809,29 +2809,45 @@ fn fn_call_kw(i: &mut TokenSlice) -> PResult<Node<CallExpressionKw>> {
     let _ = open_paren.parse_next(i)?;
     ignore_whitespace(i);
 
+    #[allow(clippy::large_enum_variant)]
+    pub enum ArgPlace {
```

Comment:
nit: `pub` here is useless (surprised this doesn't trigger a clippy lint actually)